frontend/src/components/editable/EditableAccount.vue

```diff
@@ -0,0 +1,276 @@
+        </v-card>
+      </v-menu>
+  </div>
+</template>
+
+<script>
+import AccountAvatar from '@/components/AccountAvatar'
+import ErrorMessage from './ErrorMessage'
+
+export default {
+  name: 'editable-account',
+  components: {
+    ErrorMessage,
+    AccountAvatar
+  },
+  props: {
+    items: {
+      type: Array
+    },
+    value: {
+      required: true
+    },
+    save: {
+      type: Function,
+      default: () => {}
+    },
+    rules: {
+      type: Array,
+      default: () => []
+    },
+    color: {
+      type: String,
+      default: 'blue-grey darken-2'
+    },
+    noDataText: {
+      type: String
+    },
+    readOnly: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data () {
+    return {
+      contentWidth: 500,
+      timeoutId: undefined,
+      contentBounce: false,
+      active: false,
+      loading: false,
+      messages: [],
+      error: undefined,
+      lazyValue: undefined
+    }
+  },
+  computed: {
+    isActive: {
+      get () {
+        return this.active
+      },
+      set (value) {
+        if (this.$refs.content) {
+          const { width } = this.$refs.content.getBoundingClientRect()
+          this.contentWidth = width - 8
+        }
+        this.active = value
+      }
+    },
+    internalValue: {
+      get () {
+        return this.lazyValue
+      },
+      set (value) {
+        this.lazyValue = value
+      }
+    },
+    activatorColor () {
+      if (this.contentBounce) {
+        return 'success'
+      }
+      return this.color
+    },
+    activatorIcon () {
+      if (this.contentBounce) {
+        return 'mdi-check'
+      }
+      if (this.isActive) {
+        return 'mdi-close'
+      }
+      return 'mdi-pencil'
+    }
+  },
+  methods: {
+    clearMessages () {
+      this.messages = []
+    },
+    onCancel () {
+      this.internalValue = this.value
+      this.isActive = false
+    },
+    async onSave (value) {
+      this.loading = this.color
+      try {
+        await this.save(this.internalValue)
+        this.$emit('input', this.internalValue)
+        setImmediate(() => {
+          this.isActive = false
+        })
+        this.contentBounce = true
+        this.timeoutId = setTimeout(() => {
+          this.contentBounce = false
+        }, 1000)
+      } catch (err) {
+        this.messages.push(JSON.stringify([err.message, err.detailedMessage]))
+      } finally {
+        this.loading = false
+      }
+    },
+    focus () {
+      const editable = this.$refs.editable
+      if (editable) {
+        editable.focus()
+      }
+    },
+    reset () {
+      this.messages = []
+      const editable = this.$refs.editable
+      if (editable) {
+        editable.resetValidation()
+      }
+    }
+  },
+  watch: {
+    active (value) {
+      if (value) {
+        clearTimeout(this.timeoutId)
+        this.internalValue = this.value
+        this.$emit('open')
+        setTimeout(() => {
+          this.focus()
+        }, 50)
```

Comment:
also here, use setDelayedInputFocus

Reply:
Here it makes no sense to select the content since it is an autocomplete and not a text-field.

Reply:
✔️